packages/blade/tsconfig.json

```diff
@@ -7,7 +7,8 @@
       "~utils": ["src/utils"],
       "~components/*": ["src/components/*"],
       "~tokens/*": ["src/tokens/*"],
-      "~src/*": ["src/*"],
-    }
+      "~src/*": ["src/*"]
+    },
+    "moduleSuffixes": [".all", ".web", ""]
```

Comment:
While developing internally for blade we also have this special `.all` file which we only use it for `Platform.Select`

What it does is, when resolving to this `.all` file `Platform.Select<{web, native}>` will return the union of both platform types instead of giving us either web or native. This helps while developing the components since now the platform specific type checking won't throw error in vscode.

Reply:
We don't add `.native` here?

Reply:
This tsconfig is blade's tsconfig. We can't have both web&native types resolution at once. No possible. We either have to have web or native target here. 

That's why we also added that .all, Platform.Select etc utilties.

Reply:
Suggestion to maybe add some comments in the code. Me from future will thank you 😸

Reply:
Will do, good idea.